synapse/util/task_scheduler.py

```diff
@@ -149,11 +148,6 @@ async def schedule_task(
         Returns:
             The id of the scheduled task
         """
-        if action not in self._actions:
```

Comment:
This check is moved inside `_launch_task` instead, since actions only need to be registered on the background worker and may not be registered on other ones.